### DISCUSSION TITLE
disable block media elements larger than X globally

### DISCUSSION THREAD

**Discussion by cmurtheepic** [dhowe/AdNauseam#2525] [Ideas]
I love using adnauseam, but having it block all media elements for every single page I go on is really freaking annoying. And I really wish there was a way to disable that setting for every website.

**Comment by dhowe** [comment]
Have you disabled this feature in the settings ?

![image](https://github.com/dhowe/AdNauseam/assets/737638/cd9d5254-01bb-4815-8efd-0083df3ad4a2)

**Reply by cmurtheepic** [reply]
As you can see in the comment below default behavior. it says that behavior is only overridden on a per site basis. I wish for it to apply globally to every website.

**Reply by mneunomne** [reply]
@cmurtheepic could you please share with us your settings by copying the `TROUBLESHOOTING INFORMATION` in the support tab please? Then we can understand better what is your current settings. Right now I'm not following since blokcing media elements is not a default behavior in AdNauseam.

**Reply by cmurtheepic** [reply]
@mneunomne Each time I install adnauseum in strict mode it enables that whether on my laptop or two desktops as far as I've noticed.

> AdNauseam: 3.21.0
Chromium: 123
filterset (summary):
 network: 135960
 cosmetic: 166360
 scriptlet: 33641
 html: 0
listset (total-discarded, last-updated):
 added:
  adguard-cookies: 29097-33, 1d.3h.22m
  adguard-mobile-app-banners: 5114-7, 1d.3h.21m
  adguard-other-annoyances: 14105-10, 1d.3h.21m
  adguard-popup-overlays: 26991-85, 1d.3h.21m
  adguard-widgets: 2444-1, 1d.3h.21m
  easylist-annoyances: 4450-250, 6d.3h.42m
  easylist-chat: 188-15, 6d.3h.42m
  easylist-newsletters: 7146-500, 6d.3h.42m
  easylist-notifications: 2968-1471, 6d.3h.42m
  fanboy-cookiemonster: 50684-3867, 6d.3h.42m
  ublock-annoyances: 6932-229, 1h.8m Δ
  ublock-cookies-adguard: 1060-1053, 6d.3h.42m
  [1 lists not shown]: [too many]
 default:
  user-filters: 0-0, never
  adnauseam-filters: 538-2, 6d.3h.43m
  easylist: 84575-214, 1h.8m Δ
  easyprivacy: 50423-124, 1h.8m Δ
  eff-dnt-whitelist: 613-0, 1d.3h.22m
  ublock-badware: 8083-140, 1h.8m Δ
  ublock-filters: 37496-701, 1h.8m Δ
  ublock-privacy: 781-6, 1h.8m Δ
  ublock-quick-fixes: 182-4, 1h.8m Δ
  ublock-unbreak: 2250-37, 1h.8m Δ
  urlhaus-1: 8680-5, 5h.34m
filterset (user): [empty]
trustedset:
 added: [array of 6 redacted]
userSettings:
 admap: [redacted]
 firstInstall: false
 hidingAds: true
 clickingAds: true
 blockingMalware: true
 disableHidingForDNT: true
 disableClickingForDNT: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 466 ms (selfie)
 maxAssetCacheWait: 173 ms
 cacheBackend: browser.storage.local
